submission/notifications/src/main/resources/templates/activation.html

```diff
@@ -3,7 +3,7 @@
 <p>Dear ${USERNAME}</p>
 <p>You have received this email because you have signed up for a BioStudies database account. To activate your account
   please click on the following link:<br/>
-  <a href="${URL}">Activation link</a>
+  <a href="${URL}">${URL}</a>

```

Comment:
why do we want to change title and display an url directly?

Reply:
As stated in the ticket, non HTML based e-mail clients won't render the link correctly